resources/main.mjs

```diff
@@ -234,28 +238,36 @@ function showSection(sectionIdentifier, pushState) {
         history.pushState({section: sectionIdentifier}, document.title);
 }
 
-function showHome() {
-    showSection('home', true);
-}
-
-function startTest() {
-    if (startBenchmark())
-        showSection('running');
+function showResultDetails(e) {
+    showSection('detailed-results', true);
 }
 
-function showResultsSummary() {
+function showResultsSummary(e) {
     showSection('summarized-results', true);
 }
 
-function showResultDetails() {
-    showSection('detailed-results', true);
-}
+function prepareUI() {
+    if (benchmarkClient.prepareUI)
+        benchmarkClient.prepareUI();
 
-function showAbout() {
-    showSection('about', true);
+    document.getElementById("logo").onclick = () => {
+        showSection('home', true);
+    };
```

Comment:
I can't comment on the proper lines, but see L42-46 of this file, there is still logic bound to the "logo-link" id, which was changed to "logo" here. If I understand correctly, this was preventing clicks on the logo while the tests are running.

We need to update those references, and maybe preserve the behavior with the new "logo" element.

Reply:
You're right.
I've now moved more code to address this.